test/api/buffer/exports/create/003-exists-409/request.json

```diff
@@ -0,0 +1,20 @@
+{
+  "path": "/v1/receivers/my-receiver/exports",
+  "method": "POST",
+  "headers": {
+    "Content-Type": "application/json",
+    "X-StorageApi-Token": "%%TEST_KBC_STORAGE_API_TOKEN%%"
+  },
+  "body": {
+    "name": "my-export-1",
```

Comment:
Tady jen prosím tě doplň raději to `exportId`. Dohledal jsem že se to generuje z `name`, ale ať to neplete.

Reply:
:+1: bolo by dobre skontrolovat aj to, ze vsade, kde je aj `id` aj `name` tak nie su rovnake, napr. `my-export` a `My Export` - ze sa dobre otestuje to generovanie.

Reply:
Když tam bude specifikovaný name i id tak se id nevygeneruje, takže možná spíš všude nechat jen `name`?

Reply:
Skor som myslel, ze name nedas `my-receiver`, ale `My Receiver`, a vo vystupe potom kontrolujes oba.

Reply:
Lebo ked das name priamo v takej podobe ako vygenerovane ID, tak vlastne to generovanie neoveris.